core/src/main/scala/com/metamx/tranquility/druid/DruidEnvironment.scala

```diff
@@ -24,7 +24,7 @@ class DruidEnvironment(
 ) extends Equals
 {
   // Replace / with : just like Druid does.
-  val indexService = indexServiceMaybeWithSlashes.replace('/', ':')
+  val indexServiceKey: String = indexServiceMaybeWithSlashes.replace('/', ':')
```

Comment:
Would it be possible to put a comment to where druid does this (with commit hash in url) so that future maintainers can have SOME hope of tracking what to do if this ever changes?